internal/pkg/rules/rules.go

```diff
@@ -0,0 +1,143 @@
+package rules
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	y2j "github.com/ghodss/yaml"
+	"github.com/grupozap/aegir/internal/pkg/utils"
+	livr "github.com/k33nice/go-livr"
+	"github.com/tidwall/gjson"
+	"gopkg.in/yaml.v2"
+)
+
+type RulesList struct {
+	Rules []*Rule `yaml:"rules"`
+}
+
+type Rule struct {
+	Name                     string           `yaml:"name"`
+	Namespace                string           `yaml:"namespace"`
+	ResourceType             string           `yaml:"resource_type"`
+	RulesDefinitions         []RuleDefinition `yaml:"rules_definitions"`
+	SlackNotificationChannel string           `yaml:"slack_notification_channel,omitempty"`
+}
+
+type RuleDefinition struct {
+	Field    string     `yaml:"field"`
+	LivrRule RuleObject `yaml:"livr_rule"`
+}
+
+type RuleObject struct {
+	Description string          `yaml:"description"`
+	RuleObj     livr.Dictionary `yaml:"rule"`
+}
+
+type Resource struct {
+	Kind     map[string]interface{} `json:"kind"`
+	Metadata map[string]interface{} `json:"metadata"`
+	Spec     map[string]interface{} `json:"spec"`
+}
+
+var ruleStore = map[string][]*Rule{}
+
+func RulesLoader(fp string) RulesList {
+	file, err := ioutil.ReadFile(fp)
+	if err != nil {
+		log.Printf("could not read file: %q", err)
+	}
+	var rules RulesList
+	err = yaml.Unmarshal(file, &rules)
+	if err != nil {
+		log.Printf("err: %v\n", err)
+		panic(err)
```

Comment:
If you use `log.Fatalf` you don't need to use panic because inside `Fatal` it calls `panic`.